pages/contact.html

```diff
@@ -98,8 +98,11 @@ <h4><a href="#ml">Mailing lists</a></h4>
                 <li><a href="https://lists.opensuse.org/archives/list/translation@lists.uyuni-project.org/" target="_blank">translation</a>: To discuss about translations (anyone can post as long as it is registered).</li>
             </ul>
             <h4><a href="#bugs">Reporting bugs</a></h4>
-            <p> Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
-            <div class="alert alert-warning" role="alert"> Verify that a duplicate of the issue does not already exist! </div>
+            <p>Submit bugs to the issues tab located at our <a href="https://github.com/uyuni-project/uyuni/issues">Github repository</a>.</p>
+            <p>Please verify that a duplicate of the issue does not already exist!</p>
+            <h4><a href="#bugs">Uyuni Community Hours</a></h4>
+            <p>Every last Friday of the month, 4 PM CET (Germany time) we meet online to talk about what is new in Uyuni, discuss how to improve and enhance it, answer questions from the community and have a nice time in general.</p>
```

Comment:
Personal preference: 

`Every last Friday of the month, 16.00 CET/CEST (Germany time)

Reply:
Done